rachellegelden/Home-Together-Canada
# Htc 366/member search controller

# [366](https://github.com/rachellegelden/Home-Together-Canada/issues/366)

## Summary
Added new component with controls for iterating across pages of profile results. Includes controls for adjusting page size. Changed SearchResultsContainer to use controls. Added snapshot test. Updated prior tests

## Relevant Motivation & Context
Needed in order to cycle across results which need multiple pages to display

## Testing Instructions
confirm tests pass, 1 new, 3 updated
inspect code
run server/client, sign in as member, view search page, test navigation buttons

## Developer checklist prior to opening this pull request:


## Reviewer
